### PR TITLE
Bottom line add in contact me form

### PR DESCRIPTION
**Pull request by amrendrakind**
black line added in bottom of form

### CODE REVIEW

File: portfolio.html [hunk 1]
```diff
@@ -334,6 +334,7 @@ <h3>LET'S CONNECT</h3>
           Get It Touch
         </button>
       </form>
+      <div class="bottomline"></div>
     </section>
```
Comment (gemmen29) [inline portfolio.html:338]: Kindly try to match the background image with the Figma design in the contact me section
| Your design | Figma design |
| --- | --- |
| screencapture-raw-githack-amrendrakind-MyPortfolio-MV-feature-portfolio-html-2022-02-22-20_02_21 | Template 2-Mobile-Main (1) |

the bottom shape does not appear.

File: portfolio.html [hunk 2]
```diff
@@ -334,6 +334,7 @@ <h3>LET'S CONNECT</h3>
           Get It Touch
         </button>
       </form>
+      <div class="bottomline"></div>
```
Comment (gemmen29) [inline portfolio.html:337]: Please change `Contact me` to be any heading from `h2` to `h6`, for example: `h2` instead of `div` tag for semantic purposes.

Comment (gemmen29) [inline portfolio.html:337]: Please change the text after the `Contact me` to be `p` tag instead of the `div` tag for semantic purposes.

**Review by gemmen29 — CHANGES_REQUESTED** [review]
Hi @amrendrakind, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._



## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Deepakdanger — APPROVED** [review]
Hi @Team, 

### Status: Approved ✔️

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚢 
Congratulations! 🎉 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- I would highly recommend adding project description in the README file so that any user or recruiter can easily understand your project.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**